pages/index.vue

```diff
@@ -1885,54 +1883,33 @@ export default {
       if (this.requestType === "JavaScript XHR") {
         const requestString = [];
         requestString.push("const xhr = new XMLHttpRequest()");
-        const user =
-          this.auth === "Basic Auth" ? "'" + this.httpUser + "'" : null;
+        const user = this.auth === "Basic Auth" ? `'${this.httpUser}'` : null;
         const pswd =
-          this.auth === "Basic Auth" ? "'" + this.httpPassword + "'" : null;
+          this.auth === "Basic Auth" ? `'${this.httpPassword}'` : null;
```

Comment:
Ig we can go with the `if/else` approach here. Also regarding the names, `pswd` doesn't sound that good.